style.css

```diff
@@ -0,0 +1,67 @@
+body {
+  background-color: #eee;
+  font-family: sans-serif;
+  padding-top: 200px;
+}
+
+header {
+  position: fixed;
+  top: 0%;
+  height: 180px;
+  width: 96%;
+  background-color: #eee;
+}

```

Comment:
This is an example inline comment. :slightly_smiling_face:

You can see how it's under the main summary above and point to a specific part of the code where you want to suggest changes or just comment on it generally. :wink:

To do this, make sure you go to the "File changes" tab and use the "Start a review" button for your first review comment.